packages/vm/src/cache.rs

```diff
@@ -100,6 +102,9 @@ pub struct AnalysisReport {
     /// `true` if and only if all [`REQUIRED_IBC_EXPORTS`] exist as exported functions.
     /// This does not guarantee they are functional or even have the correct signatures.
     pub has_ibc_entry_points: bool,
+    /// `true` if and only if all [`REQUIRED_IBC_V2_EXPORTS`] exist as exported functions.
```

Comment:
This references REQUIRED_IBC_V2_EXPORT**S**, but the constant is named REQUIRED_IBC_V2_EXPORT